firebase_apis/test_api/src/main/java/com/google/api/services/testing/model/TestMatrix.java

```diff
@@ -123,6 +123,16 @@
   @com.google.api.client.util.Key
   private String timestamp;
 
+  /**
+   * If true, only a single attempt at most will be made to run each execution/shard in the matrix.
+   * Flaky test attempts are not affected. Normally, 2 or more attempts are made if a potential
+   * infrastructure issue is detected. This feature is for latency sensitive workloads. The
+   * incidence of execution failures may be significantly greater for fail-fast matrices and
+   * support is more limited because of that expectation.
+   */
+  @com.google.api.client.util.Key
+  private boolean failFast;
+
```

Comment:
Did you modify this? Or is it auto generated from the API description?

Reply:
Addressed in https://github.com/Flank/flank/pull/1371